src/ansys/fluent/core/solver/flobject.py

```diff
@@ -1068,8 +1064,35 @@ def __setitem__(self, name: str, value):
         child.set_state(value)
 
 
+_items_with_string_and_allowed_values = set()
+
+
+def _filter_items_with_string_and_allowed_values(static_info):
+    """Get strings with flag has_allowed_values=True"""
+    for key, value in static_info.items():
+        if isinstance(value, dict):
+            if (
+                "type" in value
+                and "has_allowed_values" in value
+                and value["type"] == "string"
```

Comment:
@hpohekar are we also dealing with string lists?

Reply:
No.